handbook/marketing/content/editorial.md

```diff
@@ -113,5 +123,9 @@ Dan Luu summarized some [common themes from his own blog](http://danluu.com/prog
 - [Computers will have faults](http://danluu.com/postmortem-lessons/), [and systems must be designed to account for that](http://danluu.com/limplock/)
 - [Is it just me](http://danluu.com/broken-builds/), [or is stuff really broken?](http://danluu.com/everything-is-broken/)
 
+#### Brainstorm board
+
+There are a lot of ideas on our [Miro board](https://miro.com/app/board/o9J_l9gAUM4=/). Feel free to use it for inspiration or let us know if you'd like to write about one of the topics! (Note: You may have be asked to request access to the board if you don't have Miro set up with your Sourcegraph email address yet.)
```

Comment:
"Have" seems to be a typo here: "Note: You may have be asked to request access to the board"

Reply:
Thanks, fixing! https://github.com/sourcegraph/about/pull/4247